examples/graph_data_demo.ipynb

```diff
@@ -139,17 +174,168 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "source": [
+    "## Save and Load\n",
+    "Below you will see an example of how a Graph Profile can be saved and loaded again."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# The default save filepath is profile-<datetime>.pkl\n",
+    "profile.save(filepath=\"profile.pkl\")\n",
+    "\n",
+    "new_profile = dp.GraphProfiler.load(\"profile.pkl\")\n",
+    "new_report = new_profile.report()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "pp.pprint(report)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Diff\n",
+    "If we wanted to ensure that this new profile was the same as the previous profile that we loaded, we could compare them using the diff functionality."
```

Comment:
maybe can update this to have more of a story like:

If we

>  wanted to ensure that this new profile was the same as the previous profile that we loaded, we could compare them using the diff functionality.
> 
> Such reports can provide details on the differences between training and validation data like in this pseudo example:
> ```
> profiler_training = dp.Profiler(training_data)
> profiler_testing = dp.Profiler(testing_data)
> 
> validation_report = profiler_training.diff(profiler_testing)
> ```

Reply:
Added in some context to make it more meaningful